linux_os/guide/system/logging/journald/socket_systemd-journal-remote_disabled/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+title: 'Disable systemd-journal-remote Socket'
+
+description: |-
+    Journald supports the ability to receive messages from remote hosts,
+    thus acting as a log server. Clients should not receive data from
+    other hosts.
+    NOTE:
+        The same package, systemd-journal-remote , is used for both sending
+        logs to remote hosts and receiving incoming logs.
+        With regards to receiving logs, there are two Systemd unit files;
+        systemd-journal-remote.socket and systemd-journal-remote.service.
+
+rationale: |-
+    If a client is configured to also receive data, thus turning it into
+    a server, the client system is acting outside it's operational boundary.
+
+severity: medium
+
+references:
+    cis@ubuntu2204: 4.2.1.1.4
+
+ocil_clause: 'the systemd-journal-remote socket is running'
+
+ocil: |-
+    {{{ ocil_service_enabled(service="systemd-journal-remote.socket") }}}
```

Comment:
I am afraid this macro is not suitable for this case. It is suitable for services. The problem is that it mixes terminology - you put there the parameter with .socket extension, but the text provided by macro talks about services. Sockets are not services - they are Systemd units, but AFAIK socket is not a service nor subset of service. Socket is used to activate a service as a reaction to connection.
More info here:

https://unix.stackexchange.com/questions/159462/what-is-systemds-target-service-and-socket


You can inspect the final result of macro expansion by building your product and then looking into
```
build/<product>/rules/socket_systemd-journal-remote_disabled.yml
```
I think you will have to create a new macro or input the ocil manually.

Reply:
I didn't create one but instead used the available:
socket_disabled_check_with_systemd